Documentation/ceph-pool-crd.md

```diff
@@ -72,6 +72,8 @@ When creating an erasure-coded pool, it is highly recommended to create the pool
 with the default of `host`. For example, if you have replication of size `3` and the failure domain is `host`, all three copies of the data will be
 placed on osds that are found on unique hosts. In that case you would be guaranteed to tolerate the failure of two hosts. If the failure domain were `osd`,
 you would be able to tolerate the loss of two devices. Similarly for erasure coding, the data and coding chunks would be spread across the requested failure domain.
+<br>**NOTE:** Neither Rook nor Ceph will prevent the user from creating a cluster where data (or chunks) cannot be replicated safely;
+it is Ceph's design to only hang when data is written in case more OSDs are configured before the time they are required during a write.
```

Comment:
what about this for the second sentence? "It is Ceph's design to delay checking for OSDs until a write request is made. If there are not sufficient OSDs to satisfy the request, the write will hang."

Reply:
Added this terminology with some rewording to keep the note one sentence.